### ISSUE TITLE
Error when trying to get port mapping where none exists

### ISSUE THREAD

**Issue by lumaxis** [mesosphere/marathon-lb#485]
I'm trying to update to marathon-lb `1.10.0` from `1.8.0`. During startup, I'm running into this error:

```
2017-08-05 12:34:37,017 marathon_lb: Unexpected error!
Traceback (most recent call last):
  File "/marathon-lb/marathon_lb.py", line 1517, in do_reset
    self.__apps = get_apps(self.__marathon)
  File "/marathon-lb/marathon_lb.py", line 1414, in get_apps
    task_ip, task_ports = get_task_ip_and_ports(app, task)
  File "/marathon-lb/utils.py", line 407, in get_task_ip_and_ports
    task_ports = get_app_task_ports(app, task, mode)
  File "/marathon-lb/utils.py", line 385, in get_app_task_ports
    return get_port_mapping_ports(app)
  File "/marathon-lb/utils.py", line 364, in get_port_mapping_ports
    for p in port_mappings
TypeError: 'NoneType' object is not iterable
```

From what I can see, one of the apps marathon-lb is pulling does not actually have port mappings. Is this not a supported scenario, would I need to change anything else? Thanks!

**Comment by waz0wski** [comment]
recently ran into this  and seems this issue was resolved in [v1.10.2](https://github.com/mesosphere/marathon-lb/releases/tag/v1.10.2)

@matt-deboer

**Comment by jkoelker** [comment]
Confirmed fixed in commit https://github.com/mesosphere/marathon-lb/commit/0a5906fa6401fa1c7cecce29b6775a59446a3019#diff-04d2c9518498da64bfa9db44c6211645R343.
